ads/purch.js

```diff
@@ -25,7 +25,7 @@ import {
  */
 export function purch(global, data) {
   validateData(data, [],
-      ['pid', 'divid']);
+      ['pid', 'divid', 'config']);
```

Comment:
Thank you. One request, could you please update the documentation. And indicate that `data-config` is also supported. 
And since the param is optional, I'll let you decide if you want to change the example or not. Thanks.

Reply:
Yuxuan,
I added data-config parameter in the documentation. Thank you.
Do you know when is the next AMP release?

Reply:
we cut every Tuesday. And change goes to production one week after.